ext/storage_sources/azure_store/azure_connection.cpp

```diff
@@ -44,9 +46,15 @@ azure_connection::list_objects(std::vector<std::string> &objects) const
     return 0;
 }
 
+/* 
+ * Puts an object (blob) given a reference to the filename
+ */
 int
 azure_connection::put_object(const std::string &file_name) const
 {
+    std::filesystem::path p(file_name);
+    Azure::Storage::Blobs::BlockBlobClient blobClient = _azure_client.GetBlockBlobClient(p.filename());
+    Azure::Response<Azure::Storage::Blobs::Models::UploadBlockBlobResult> result = blobClient.UploadFrom(file_name);
```

Comment:
ditto and is it possible that the result fails? If so I would like to return -1 here.

Reply:
use of auto resolved here https://github.com/wiredtiger/wiredtiger/pull/8718/commits/ed135883a94c5734254cb1154062cb548ec3a29e

Reply:
Thanks for fixing the first part, can we address the second part?

Reply:
Looking at the SDK implementation I'm not too sure what would be the best way to go forward with this. The `UploadFrom()` function will always either create a new file or update the file if the file already exists. It returns a `UploadBlockBlobResult` which has a couple fields listed here: https://azuresdkdocs.blob.core.windows.net/$web/cpp/azure-storage-blobs/12.6.2/struct_azure_1_1_storage_1_1_blobs_1_1_models_1_1_upload_block_blob_result.html but again nothing like bool to help make things easier to comprehend

Reply:
I have removed the result value from UploadFrom as it does not return anything useful and will always work given a valid file_name. In addition Object exist will need to be implemented and will prevent accidental updates to an existing Block Blob in the container with the same name